### DISCUSSION TITLE
What does the component in mantine.dev website docs consist of?

### DISCUSSION THREAD

**Discussion by asyncink** [mantinedev/mantine#6035] [Q&A]
Hello guys! Thank you for for your ui lib, it looks and feels awesome so far!

I'm curious, what blocks does this component consist of in the documentation website?

<img width="901" src="https://github.com/mantinedev/mantine/assets/24447057/3698d2b4-2150-4f2e-a4f3-fb51df002c3d">

I mean main component that is divided to three parts: preview, controls and source code. It is simple but it looks like it can be built without any custom css? I tried Box and Paper, but without my own styles it doesn’t fit example from the screenshot.

Thank you for your support and have a good day!

**Comment by rtivital** [comment]
The website is open source, you can find its code in the main repository

**Reply by asyncink** [reply]
I started this night, but docs sources are not as straightforward as I thought, so I’d opened this discussion — with a chance the question is easier to answer than dig into sources.

**Reply by asyncink** [reply]
Will post answer here (if none will appear) after reviewing docs sources.
